### PR TITLE
Allow creation of empty mask

### PR DESCRIPTION
**Pull request by valosekj**
This PR adds a feature that creates an empty mask if segmentation does not exist. This is useful, for example, when creating MS lesion segmentations from scratch. 

Please, could you double-check especially the part of the code responsible for empty mask creation:
https://github.com/spinalcordtoolbox/manual-correction/blob/bba77adbb97f28fa68c5c13a4b81e69a8e547331/utils.py#L254-L264

Fixes https://github.com/spinalcordtoolbox/manual-correction/issues/4

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 import shutil
 import yaml
 
+import numpy as np
```
Comment (sandrinebedard) [inline utils.py:19]: It is not included in `requirements.txt`

File: utils.py [hunk 2]
```diff
@@ -16,6 +16,9 @@
...
+import nibabel as nib
```
Comment (sandrinebedard) [inline utils.py:20]: It is not included in `requirements.txt`

**Review by sandrinebedard — CHANGES_REQUESTED** [review]
Just need to add the added requirements, the rest works well!

I tested for `FILES_LESION` and `FILES_LABEL` and works perfectly.

File: utils.py [hunk 3]
```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
             logging.error("'{}' is not installed. Please install it before using this program.".format(software))
             install_ok = False
     return install_ok
+
+
+def create_empty_mask(fname, fname_label):
+    """
+    Create empty mask from reference image
+    :param fname: path to reference image
```
Comment (naga-karthik) [inline utils.py:257]: Is this the absolute path? If so, then explicitly saying it as "absolute path to reference image" would be clear

File: utils.py [hunk 4]
```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
...
+    :param fname_label: path to output mask under derivatives
+    """
+    img = nib.load(fname)
+    data = np.zeros(img.shape)
+    img_mask = nib.Nifti1Image(data, img.affine, img.header)
```
Comment (naga-karthik) [inline utils.py:262]: While it is implied, I find it better to add the argument name also, i.e 
```
img_mask = nib.Nifti1Image(data, affine=img.affine, header=img.header)
```
You don't _have_ to do this but it just makes things more explicit

File: utils.py [hunk 5]
```diff
@@ -246,3 +249,16 @@ def check_software_installed(list_software=['sct']):
...
+    nib.save(img_mask, fname_label)
+    print("Empty mask created: {}".format(fname_label))
```
Comment (naga-karthik) [inline utils.py:264]: Maybe edit this to say `Empty mask created at: {}".format(fname_label)`

**Review by naga-karthik — APPROVED** [review]
Minor suggestions, rest looks good!

File: utils.py (hunk 1 above)
Comment (valosekj) [reply utils.py:19]: Good catch! I forgot to push this change. Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/06dba920ec84786573596de77cd4da943b1c8e8f

File: utils.py (hunk 2 above)
Comment (valosekj) [reply utils.py:20]: Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/06dba920ec84786573596de77cd4da943b1c8e8f

File: utils.py (hunk 3 above)
Comment (valosekj) [reply utils.py]: Good point! Addressed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785

File: utils.py (hunk 4 above)
Comment (valosekj) [reply utils.py]: Great suggestion! Done in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785

File: utils.py (hunk 5 above)
Comment (valosekj) [reply utils.py]: Okay! Done in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/b1f341c420f073350ad2bd45184af8a6f6e03785

**Comment by valosekj** [comment]
Thank you for reviewing! I addressed your comments. --> merging

**Pull request merged**